walker.go

```diff
@@ -19,9 +19,29 @@ type WalkOpt struct {
 	// FollowPaths contains symlinks that are resolved into include patterns
 	// before performing the fs walk
 	FollowPaths []string
-	Map         FilterFunc
+	Map         WalkFunc
 }
 
+type WalkFunc func(string, *types.Stat) WalkResult
```

Comment:
I guess this is more like a `FilterFunc/FilterResult`

Reply:
ya, the problem is that there's already a FilterFunc
https://github.com/tonistiigi/fsutil/blob/master/diskwriter.go#L30
and the places it's used, Walk/Skip doesn't really make sense

maybe we could call it WalkFilterFunc / WalkFilterResult?

Reply:
Maybe just `MapFunc` then

Reply:
:+1: , updated!